app/controllers/volunteers_controller.rb

```diff
@@ -72,6 +79,7 @@ def set_volunteer
 
   def volunteer_params
     params.require(:volunteer).permit(volunteer_attributes, :rejection_type, :rejection_text,
-      :trial_period, :intro_course, :doc_sent, :bank_account, :evaluation)
+      :trial_period, :intro_course, :doc_sent, :bank_account, :evaluation, :flexible,
```

Comment:
I guess that was was because it wasn't rebased yet. Now everything seems to be in order with those... 👍